syntax = "proto3";

package framework.rpc;
import "ccsrc/adapters/tensorflow/rpc/graph.proto";

message CallRequest {
    Graph graph = 1;
    string policy = 2;
}

message CallResponse {
    Graph graph = 1;
    bool success = 2;
}

service RpcService {
    // RPC for invoking a registered function on remote server.
    rpc Call(CallRequest) returns (CallResponse) {}
}
